R/plot-observed-vs-simulated.R

```diff
@@ -81,6 +84,11 @@ plotObservedVsSimulated <- function(dataCombined,
   # - For logarithmic scale: `0`
   defaultFoldDistance <- ifelse(is_any_scale_linear, 0, 1)
 
+  # foldDistance should not be lower than 1
+  if (any(foldDistance < 1)) {
+    stop(messages$plotObservedVsSimulatedWrongFoldDistance("foldDistance", foldDistance))
```

Comment:
I think fold distance = 1 does not make sense as well

Reply:
Restricted to >1.